{
  "type": "class",
  "name": "CommandException",
  "qualified_name": "net.minecraft.command.CommandException",
  "package": "net.minecraft.command",
  "modifiers": [],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": []
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "CommandException",
      "modifiers": [],
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [
    {
      "name": "getErrorObjects",
      "modifiers": [
        "public"
      ],
      "return_type": "",
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.8.9/net/minecraft/command/CommandException.html"
}